projects/yel/index.js

```diff
@@ -0,0 +1,106 @@
+const sdk = require("@defillama/sdk");
+const Result = require("./Helpers/Result");
+const YelLPFarm = require("./Providers/YelLPFarm");
+const addr = require("./config/addresses.json");
+const YelEnhancedLpFarm = require("./Providers/YelEnhancedLpFarm");
+const YelSingleStaking = require("./Providers/YelSingleStaking");
+const YelEnhancedSingleStake = require("./Providers/YelEnhancedSingleStake");
+
+async function ethereumTvl() {
+    const [
+        yelInLpEth,
+        yelInLpBsc,
+        yelInLpFantom,
+        yelInLpPolygon,
+        wethInLp,
+        ifarmEnhanced,
+        yelSSEth,
+        yelSSBsc,
+        yelSSFantom,
+        yelSSPolygon,
+    ] = await Promise.all([
+        YelLPFarm.yelTokensInLp(addr.yelFarmingContract.ethereum, 'ethereum'),
+        YelLPFarm.yelTokensInLp(addr.yelFarmingContract.bsc, 'bsc'),
```

Comment:
Because coingecko did not added contract addresses for side chains and sdk cant find token on other chains. Thats wy we had to place all YEL tokens on the ethereum.